src/gwt/src/org/rstudio/core/client/widget/ModalDialogBase.java

```diff
@@ -95,6 +101,7 @@ protected ModalDialogBase(SimplePanel containerPanel, DialogRole role)
       bottomPanel_.add(ariaLiveStatusWidget_);
 
       setButtonAlignment(HasHorizontalAlignment.ALIGN_RIGHT);
+      mainPanel_.add(anchorPanel_);
```

Comment:
Do we want to be "smarter" about whether to always add this or only add it when the `addLink` is called or internally when `addAnchorWidget` is called?

Reply:
That's what I wanted to do but it wasn't really possible due to the multiple levels of abstractions involved and the way it all gets wired up. I think it's fine to add an element then remove it (have to do that otherwise the visual layout changes for dialogs that don't have the link).

Reply:
I mean, there would be some way to make it happen, I'm sure, but it seemed more trouble than it was worth.